extensions/ql-vscode/test/pure-tests/databases/db-config-store.test.ts

```diff
@@ -17,39 +17,37 @@ describe('db config store', async () => {
   });
 
   it('should create a new config if one does not exist', async () => {
-    const configPath = path.join(tempWorkspaceStoragePath, 'workspace-databases.json');
-
     const configStore = new DbConfigStore(tempWorkspaceStoragePath, extensionPath);
     await configStore.initialize();
 
-    expect(await fs.pathExists(configPath)).to.be.true;
```

Comment:
Why have we removed this check? I think it's good to check the file is actually created.

Reply:
Good catch, thanks! Will revert it back